aspnetcore/grpc/aspnetcore.md

```diff
@@ -98,24 +115,38 @@ For more information on enabling HTTP/2 and TLS with Kestrel, see [Kestrel endpo
 > [!NOTE]
 > macOS doesn't support ASP.NET Core gRPC with TLS. Additional configuration is required to successfully run gRPC services on macOS. For more information, see [Unable to start ASP.NET Core gRPC app on macOS](xref:grpc/troubleshoot#unable-to-start-aspnet-core-grpc-app-on-macos).
 
+## IIS
+
+[Internet Information Services (IIS)](xref:host-and-deploy/iis/index) is a flexible, secure and manageable Web Server for hosting web apps, including ASP.NET Core. .NET 5 and Windows 10 Build 20241 or later are required to host gRPC services with IIS.
+
+IIS must be configured to use TLS and HTTP/2. For more information, see <xref:host-and-deploy/iis/protocols>.
+
+## HTTP.sys
+
+[HTTP.sys](xref:fundamentals/servers/httpsys) is a web server for ASP.NET Core that only runs on Windows. .NET 5 and Windows 10 Build 20241 or later are required to host gRPC services with HTTP.sys.
```

Comment:
Didn't Http.Sys start working with an earlier version than IIS? I have 19529 referenced in the tests. I guess it doesn't matter if that version is still greater than the currently released 20H2 (19042).

https://github.com/dotnet/aspnetcore/blob/6f26dee9a9b43b5cc0f56542ef8ba2df16ee376a/src/Servers/HttpSys/test/FunctionalTests/Http2Tests.cs#L25